### ISSUE TITLE
neural network model with batch normalization

### ISSUE THREAD

**Issue by minseok-ryu** [APPFL/APPFL#78]
When evaluating a neural network model with a batch normalization (e.g., resnet) in APPFL, the testing accuracy was different from the one obtained by PyTorch. It turns out that the ``running mean/variance`` had not been applied for evaluation, which is used for normalization during the evaluation as indicated in  

```https://discuss.pytorch.org/t/what-does-model-eval-do-for-batchnorm-layer/7146```

Note the difference between ``state_dict()`` and ``parameters()``as indicated in 

```https://discuss.pytorch.org/t/difference-between-state-dict-and-parameters/37531```
